Modules/HR/Http/Controllers/EmployeeController.php

```diff
@@ -28,10 +28,24 @@ public function __construct(EmployeeService $service)
      */
     public function index(Request $request)
     {
+        $this->authorize('list', Employee::class);
+        $search = request()->query('employeename') ?: '';
+        $employeeData = Employee::with('employees');
         $filters = $request->all();
         $filters = $filters ?: $this->service->defaultFilters();
         $name = request('name');
-        $employeeData = Employee::where('staff_type', $name)->get();
+        $employeeData = Employee::where('staff_type', $name)
+            ->leftJoin('project_team_members', 'employees.user_id', '=', 'project_team_members.team_member_id')
+            ->selectRaw('employees.*, team_member_id, count(team_member_id) as Count')
```

Comment:
Lets use more meaningful name
```suggestion
            ->selectRaw('employees.*, team_member_id, count(team_member_id) as project_count')
```